.github/workflows/ci.yml

```
@@ -87,7 +87,7 @@ jobs:
     - name: Build addons
       run: |
         cd grass-addons/src
-        GRASS_INSTALL=$HOME/install/grass*
+        GRASS_INSTALL=$HOME/install/grass
```

For the current development version the installation base directory is grass83, thus CI is still failing.

The approach below should catch that (works in my fork)...

```suggestion
        GRASS_INSTALL=$($HOME/install/bin/grass --config | sed -n '4{p;q}')
```

---

Instead of hard-coding GRASS versions, we could take it from grass --config (see above)...

---

Yes you are right. Thanks for hint!